src/Fetch.js

```diff
@@ -21,6 +21,7 @@ var Fetch = {
   },
 
 #if FETCH_SUPPORT_INDEXEDDB
+  // Be cautious that `onerror` may be run synchronously
```

Comment:
Another option might be to make the onerror async (add a setTimeout of 0)?

Reply:
Good point, I just made that change.